input/script_utils.go

```diff
@@ -203,35 +203,40 @@ func Ripemd160H(d []byte) []byte {
 // output payment for the sender's version of the commitment transaction. The
 // possible script paths from this output include:
 //
-//    * The sender timing out the HTLC using the second level HTLC timeout
-//      transaction.
-//    * The receiver of the HTLC claiming the output on-chain with the payment
-//      preimage.
-//    * The receiver of the HTLC sweeping all the funds in the case that a
-//      revoked commitment transaction bearing this HTLC was broadcast.
+//   - The sender timing out the HTLC using the second level HTLC timeout
+//     transaction.
+//   - The receiver of the HTLC claiming the output on-chain with the payment
+//     preimage.
+//   - The receiver of the HTLC sweeping all the funds in the case that a
+//     revoked commitment transaction bearing this HTLC was broadcast.
 //
 // If confirmedSpend=true, a 1 OP_CSV check will be added to the non-revocation
 // cases, to allow sweeping only after confirmation.
 //
 // Possible Input Scripts:
-//    SENDR: <0> <sendr sig>  <recvr sig> <0> (spend using HTLC timeout transaction)
-//    RECVR: <recvr sig>  <preimage>
-//    REVOK: <revoke sig> <revoke key>
-//     * receiver revoke
+//
+//	SENDR: <0> <sendr sig>  <recvr sig> <0> (spend using HTLC timeout transaction)
+//	RECVR: <recvr sig>  <preimage>
+//	REVOK: <revoke sig> <revoke key>
+//	 * receiver revoke
 //
 // OP_DUP OP_HASH160 <revocation key hash160> OP_EQUAL
 // OP_IF
-//     OP_CHECKSIG
+//
+//	OP_CHECKSIG
```

Comment:
Only parts of the script are formatted.

Reply:
Fixed by converting them into code blocks.